### DISCUSSION TITLE
Drop Transmission-GTK in favor of Transmission-Qt?

### DISCUSSION THREAD

**Discussion by Pentaphon** [transmission/transmission#3689] [Polls]
I don't see a lot of interest in Transmission-GTK anymore. GTK4 is out and nobody has submitted any PRs to help with this. 

Somebody even made a new GTK4 client with Transmission as a backend instead of just contributing to this project. https://gitlab.gnome.org/World/Fragments

I think at this point, for the health and future of the project, Transmission should drop GTK support and just go Qt only since that toolkit gets better support from this community and the Qt community as well. I don't think we will ever see GTK look good or modern on Linux and right now people are probably focusing more on Mac than Linux so we might as well have the Linux port go entirely Qt so that the Qt port can work for Windows and Linux at the same time to save people time and effort in the long run. 

GTK development is just slowing us down and we should drop the dead weight sooner rather than later.

**Comment by barracuda156** [comment]
If Qt4 was supported, GTK would be unneeded, but since Qt4 is no more, GTK should rather stay for compatibility.
